guides/common/attributes.adoc

```diff
@@ -97,6 +96,11 @@ ifeval::["{build}" == "foreman"]
 :ansible-namespace: `theforeman.foreman`
 :ansible-namespace-example: `theforeman.foreman._module_name_`
 :awx: AWX
+:BaseURL: https://docs.theforeman.org/master/
```

Comment:
Instead of master could we use {ProductVersion}, then define that as master upstream for now?
I think that if we are going to get serious about replacing Foreman manuals, we need to start branching.
I think that we should think seriously about this in the next month because we need the process worked out before @spetrosi leaves the docs team.

Reply:
Let's start branching after we migrate installation guide. I was hoping to get to it soon.

Reply:
We can modify `master` later tho, I won't block the PR.

Reply:
Yeah for sure, but as part of this PR I'd like {ProductVersion} used to define the upstream version, so that it is then set and we don't have to modify that again.

Reply:
Let's do this as part of the issue #322 after the issue #192 is resolved. I plan to finish with #192 by the end of January.

Reply:
OK.

Reply:
lol I didn't reload this page for hours and did not see your comments prior to adding mine. The issue is that ProductVersion is now defined globally for all builds, so we will need to move this to each builds' attributes. Then, there is a bunch of similar attributes like ProductVersionPrevious, that are very related and that I want to keep all in one place, so we will need to move the altogether and find new names for them in each build. This is a completely different work from what this PR relates to. Let's shoot at all those moving targets one by one rather than fixing them all here.

Reply:
I'm happy with this as long as they are defined somewhere, like in Issues, so we can look at the upstream implications also.

Reply:
Thanks, @melcorr, can you merge this PR now that it is approved?